Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.8 AS conpot-builder
+
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN adduser --disabled-password --gecos "" conpot
+
+RUN mkdir /opt/conpot
+COPY . /opt/conpot/
+RUN chown conpot:conpot -R /opt/conpot
```

Comment:
COPY has a form that does the `chown` for you. The run container uses that syntax. :man_shrugging: 

https://docs.docker.com/engine/reference/builder/#copy

Also the mkdir should be redundant:
> If \<dest\> doesn’t exist, it is created along with all missing directories in its path.

Reply:
Nice, smarter every day!